behaviours/meme_rater.py

```diff
@@ -139,28 +188,36 @@ async def rate_meme(
             curr_ratings = (0, 0)
             entry_exists = False
 
-        avg_rating = min(
-            max(
-                0, (sum(ratings) + curr_ratings[0]) // (len(ratings) + curr_ratings[1])
-            ),
+        ratings_sum = sum([result.rate for result in rating_results])
+        ratings_count = len(rating_results)
+
+        new_rating_sum = ratings_sum + curr_ratings[0]
+        new_rating_count = ratings_count + curr_ratings[1]
+
+        avg_rating: int = min(
```

Comment:
Does python not have a `clamp` function?

Reply:
No clamp function in python